magnet/excitation-data/si-id-apu22-1991a.txt

```diff
@@ -8,33 +8,43 @@
 
 # B.LAMBDA DATA
 # =============
-+0000.00  +1.540885e-02 +0.000000e+00
-+0000.50  +1.527133e-02 +0.000000e+00
-+0001.00  +1.515087e-02 +0.000000e+00
-+0001.50  +1.497548e-02 +0.000000e+00
-+0002.00  +1.470837e-02 +0.000000e+00
-+0002.50  +1.433774e-02 +0.000000e+00
-+0003.00  +1.386794e-02 +0.000000e+00
-+0003.50  +1.331215e-02 +0.000000e+00
-+0004.00  +1.268644e-02 +0.000000e+00
-+0004.50  +1.200532e-02 +0.000000e+00
-+0005.00  +1.127867e-02 +0.000000e+00
-+0005.50  +1.051018e-02 +0.000000e+00
-+0006.00  +9.697129e-03 +0.000000e+00
-+0006.50  +8.831722e-03 +0.000000e+00
-+0007.00  +7.903777e-03 +0.000000e+00
-+0007.50  +6.904887e-03 +0.000000e+00
-+0008.00  +5.834013e-03 +0.000000e+00
-+0008.50  +4.704520e-03 +0.000000e+00
-+0009.00  +3.552650e-03 +0.000000e+00
-+0009.50  +2.447429e-03 +0.000000e+00
-+0010.00  +1.502023e-03 +0.000000e+00
-+0010.50  +8.865227e-04 +0.000000e+00
-+0011.00  +8.421742e-04 +0.000000e+00
++0000.00  +1.523075e-02 +0.000000e+00
++0000.50  +1.500636e-02 +0.000000e+00
++0001.00  +1.486126e-02 +0.000000e+00
++0001.50  +1.468199e-02 +0.000000e+00
++0002.00  +1.440906e-02 +0.000000e+00
++0002.50  +1.402144e-02 +0.000000e+00
++0003.00  +1.352332e-02 +0.000000e+00
++0003.50  +1.293291e-02 +0.000000e+00
++0004.00  +1.227340e-02 +0.000000e+00
++0004.50  +1.156606e-02 +0.000000e+00
++0005.00  +1.082548e-02 +0.000000e+00
++0005.50  +1.005693e-02 +0.000000e+00
++0006.00  +9.255870e-03 +0.000000e+00
++0006.50  +8.409567e-03 +0.000000e+00
++0007.00  +7.500887e-03 +0.000000e+00
++0007.50  +6.514188e-03 +0.000000e+00
++0008.00  +5.443365e-03 +0.000000e+00
++0008.50  +4.302022e-03 +0.000000e+00
++0009.00  +3.135782e-03 +0.000000e+00
++0009.50  +2.036730e-03 +0.000000e+00
++0010.00  +1.159993e-03 +0.000000e+00
++0010.50  +7.424536e-04 +0.000000e+00
++0011.00  +1.123595e-03 +0.000000e+00
 
 # COMMENTS
 # ========
 #
-# 1. copied from su-id-apu22-1991b.
+# 1. Measurement data and analysis scripts in lnls350-linux bare repo, /home/fac_files/repos-bares/insertion_devices.git
+#    These measurements correspond to the data files in:
+#    insertion_devices/kyma_p22mm_w200mm_g08mm/1991a/fieldmaps
 #
-# NOTE: !!! needs updating !!!
+# 2. K ~ 93.3729/(T.m) * (lambda * B)
+#    This table contains lambda * B as a function of APU phase.
+#    In terms of maximum deflection angle,
+#       lambda * B == 2 * pi * Brho * x'_max
+#       where x'_max is taken from fitting RK trajectory within 40 central periods (20 before and 20 after the ID center)
+#
+#  3. Polynomial of order 6 is fitted to data in order to filter out data fluctuations and to increase grid points.
```

Comment:
@murilobalves , in this case it was not necessary to increase number of points, right? do you intend to keep the original message as as general comment?

Reply:
yes, my intention is to keep in the comments the required information to reproduce the excitation data, in this case: 
1) the data location, 
2) the script used to generate the excdata, 
3) the number of central periods and 
4) the polynomial order used in the fit

Reply:
I understand. Just as it has been done for most of other excdata files. But my question is whether we should add "to increase the number of grid points" in this excdata file in particular, since the raw data already had 23 points. (maybe you could right "in this case the number of data points were not increased in the fitting process...", just as a remainder that this is a possibility when fitting raw data to smooth them.

Reply:
I didn't get your question in the first time. Now I got it, done!